Tarea 3

H-Cote opened the pull request:
Primera versión de la tarea 3. 

Se inicia el ejercicio 1.

H-Cote committed:
Cuarta versión de la tarea 2

Se corrigen errores ortográficos y se describe la convergencia de las funciones en términos de sus pendientes.

H-Cote committed:
Quinta versión de la tarea 2

Se corrigen errores ortográficos y se modifica la respuesta a la primera pregunta del ejercicio 4.

H-Cote committed:
Actualización del repositorio

Merge branch 'master' of https://github.com/lbenet/2017-2_TSFisComputacional-1 into Tarea-2

gitrmora committed:
primer pull de la tarea 3

carlosr47 committed:
primer commit

ruben-araiza committed:
tarea 3

ruben-araiza committed:
se agrega el primer ejercicio de la tarea 3

H-Cote committed:
Primera versión de la tarea 3

Se inicia la primera parte de la tarea 3

Commit:
Agrego el notebook 08, la Tarea4, cambios en tareas/README.md y renombro el 06

lbenet commented:
Recibida. 

Como discutimos ayer en clase, `xdual` debe ser *independiente* de cualquier función `f`, ya que es lo que define a la variable independiente, que es respecto a la que derivas.

Finalmente, te hago notar que hay algún problema para incorporarlo directamente al proyecto! Me parece que la razón es que hay commits que pertenecen a la tarea anterior...

gitrmora committed:
final de la tarea3 y archivos *.jl

carlosr47 committed:
modificaciones en ejercicio1 duda en stackoberflow

gitrmora committed:
corrección final de la tarea3

gitrmora committed:
últimas correcciones al archivo runtest.jl de la tarea3

H-Cote committed:
Segunda versión de la tarea 3

Se completan los ejercicios de la tarea y se crean los archivos AutomDiff.jl y runtest.jl.

lbenet commented:
Vas bien, pero te faltan tests; por ejemplo, para `f(x) = 2*x^3-1`, funciona `f( xdual(2.5) )`? Me parece que no... Este tipo de cosas son las que aparecen en el notebook (pero que no es necesario hacer para la tarea!).

Otra cosa: hay conflictos que necesitas resolver si quieres que se acepte tu tarea.

carlosr47 committed:
avances en problema2 y corrección del problema1

carlosr47 committed:
correcciones en problema2

carlosr47 committed:
concluyendo problema 2 e implementando regla de la cadena

H-Cote committed:
Tercera versión de la tarea 3.

Se agregan nuevos test y se calcula el valor de la derivada de la función indicada

carlosr47 committed:
importanrdo codigo en automdiff.jl

carlosr47 committed:
segundo archivo runtest.jl creado, tarea3 completada

lbenet commented:
Hay algunos detalles. Por ejemplo, es incorrecto `using Base.test`; debe ser `using Base.Test`; ese error proviene de lo que yo escribí en el notebook.

Por otro lado, viendo lo que has hecho, si por ejemplo checas `Dual(1.0) == 1.0` te va a dar un error. Esto significa que tus tests no incluyen todos los casos... Por último, vale la pena que uses las posibilidades,por ejemplo de `@testset`

lbenet commented:
Sigues teniendo conflictos para hacer el merge/rebase directamente....

Commit:
Corregí una errata en el ejercicio 4 del notebook 06 y Tarea3

carlosr47 committed:
corrigiendo runtest.jl

carlosr47 committed:
corrigiendo tarea3 y automdiff.jl eliminando puntos suspensivos

carlosr47 committed:
insertando linea include(runtest.jl) en tarea3

carlosr47 committed:
comentando varios aspectos de automdiff.jl

carlosr47 committed:
usando @testset

carlosr47 committed:
revisando codigo de tarea3.ipynb

H-Cote committed:
Cuarta versión de la tarea 3

Se agregan tests y se incluyen en el archivo runtest.jl

ruben-araiza committed:
Agrego test de xdual

ruben-araiza committed:
Agrego ejercicio 2, operaciones de duales

ruben-araiza committed:
agrego archivos AutomDiff.jl y runtest.jl

ruben-araiza committed:
pequeñas correcciones

ruben-araiza committed:
agrego documentacion a funciones

lbenet commented:
La tarea no puede ser aceptada si hay conflictos. 

Sugiero que en el ejercicio 3 muestres que todo funciona, por ejemplo, incluyendo (`include`) el archivo `runtests.jl`.

lbenet commented:
Otra cosa: en `runtests.jl`, debería ser `using Base.Test`, donde "Test" es con T mayúscula.

H-Cote committed:
Quinta versión de la tarea 3

Se corrige el problema del archivo runtest.jl

H-Cote committed:
Merge branch 'Tarea_3' of https://github.com/H-Cote/2017-2_TSFisComputacional-1 into Tarea_3
Actualización del repositorio

H-Cote committed:
Mejoras a runtest.jl

lbenet commented:
Cierro este PR ya que se reenvió en #50.
